charts/mla-secrets/values.yaml

```diff
@@ -14,6 +14,8 @@
 
 
 minio:
+  ## Enable if you want to generate a minio secret.
+  enabled: false
```

Comment:
What about making it in the opposite way - enabled by default? Otherwise this may break upgrade if custom values are used.

Reply:
For this I am not sure, because we shouldn't let user install this helm chart during upgrading.
IIRC this is only used the first time you want to deploy MLA on seed, and running this multiple times will just override the secrets, and cortex and loki will have issues when they try to access minio.

Reply:
but you are right, this is definitely a breaking change, and I will update it in the opposite way

Reply:
Done